README.md

```diff
@@ -1 +1,139 @@
-# AsylumAdvice app
+<!-- PROJECT SHIELDS --> <!-- *** I'm using markdown "reference style" links for readability. *** Reference links are enclosed in brackets [ ] instead of parentheses ( ). *** See the bottom of this document for the declaration of the reference variables *** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use. *** https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+[![Contributors][contributors-shield]][contributors-url] [![Forks][forks-shield]][forks-url] [![Stargazers][stars-shield]][stars-url] [![Issues][issues-shield]][issues-url] [![MIT License][license-shield]][license-url] [![LinkedIn][linkedin-shield]][linkedin-url]
+
+<!-- PROJECT LOGO --> <br>
+
+[![Logo](public/images/asylum-advice-logo.svg)](https://github.com/AsylumAdvice/app) # Asylum Advice Asylum Advice helps asylum seekers find the information that is relevant to their asylum procedure.<br>
```

Comment:
<img width="421" alt="Screenshot 2020-03-03 at 11 31 29" src="https://user-images.githubusercontent.com/3501767/75767125-91966a00-5d42-11ea-8f79-56d4e1d260f6.png">

nit pick: this currently weirdly rendered as markdown

Reply:
I used the cli create-react-app, not expo.

Reply:
I mean to say, the cli react-native init, not expo.

Reply:
the documentation says `Follow these instructions if you need to build native code in your project. For example, if you are integrating React Native into an existing application, or if you "ejected" from Expo, you'll need this section.`

i think for our app expo would be sufficient. when i tried running `npm start` with the following log

```
 npm start

> app@0.0.1 start /Users/qaiserabbasi/git/app
> react-native start

┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│  Running Metro Bundler on port 8081.                                         │
│                                                                              │
│  Keep Metro running while developing on any JS projects. Feel free to        │
│  close this tab and run your own Metro instance if you prefer.               │
│                                                                              │
│  https://github.com/facebook/react-native                                    │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

Looking for JS files in
   /Users/qaiserabbasi/git/app

Loading dependency graph, done.

To reload the app press "r"
To open developer menu press "d"
```

there's no QR code popping up / there's nothing happening after this message. when i used `expo` i got a nice overview page with a QR code to scan and everything works perfectly

![Screenshot 2020-03-03 at 11 54 44](https://user-images.githubusercontent.com/3501767/75769103-da035700-5d45-11ea-83b0-2220e7cb77ad.png)

Reply:
ok, so shall I re-create with expo and make another pull request?